ecf/subseasonal/prep/jevs_subseasonal_prep.ecf

```diff
@@ -0,0 +1,101 @@
+#PBS -N jevs_subseasonal_prep_00
+#PBS -j oe 
+#PBS -o /lfs/h2/emc/vpppg/noscrub/shannon.shields/EVS/ecf/subseasonal/prep/evs_subseasonal_prep_out.txt
+#PBS -S /bin/bash
+#PBS -q "dev"
+#PBS -A VERF-DEV
+#PBS -l walltime=02:00:00
+#PBS -l place=vscatter:exclhost,select=1:ncpus=128:ompthreads=1
+#PBS -l debug=true
+#PBS -V
+
+set -x
+
+export model=evs
+
+cd $PBS_O_WORKDIR
+module reset
+
+export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/EVS
+
+versionfile=$HOMEevs/versions/run.ver
+. $versionfile
+module load prod_envir/2.0.6
+module load prod_util/2.0.13
+export DATAROOTtmp=/lfs/h2/emc/stmp/$USER/evs_test/$envir/tmp
+export job=${PBS_JOBNAME:-jevs_subseasonal_prep}
+export jobid=$job.${PBS_JOBID:-$$}
+export TMPDIR=$DATAROOTtmp
+export SITE=$(cat /etc/cluster_name)
+
+#%include <head.h>
+#%include <envir-p1.h>
+
+############################################################
+# Load modules
+############################################################
+module load intel/${intel_ver}
+module load PrgEnv-intel/${PrgEnvintel_ver}
+module load craype/${craype_ver}
+module load cray-mpich/${craympich_ver}
+module load cray-pals/${craypals_ver}
+module load cfp/${cfp_ver}
+module load g2c/${g2c_ver}
+module load wgrib2/${wgrib2_ver}
+module load proj/${proj_ver}
+module load geos/${geos_ver}
+export HPC_OPT=/apps/ops/prod/libs
+module use /apps/ops/prod/libs/modulefiles/compiler/intel/19.1.3.304/
+module load intel/${intel_ver}
+module load gsl/${gsl_ver}
+module load libpng/${libpng_ver}
+module load libjpeg/${libjpeg_ver}
+module load netcdf/${netcdf_ver}
+module load udunits/${udunits_ver}
+module load nco/${nco_ver}
+module load grib_util/${grib_util_ver}
+module load python/${python_ver}
+module load met/${met_ver}
+module load metplus/${metplus_ver}
+module load cdo/${cdo_ver}
+
+module list
+
+export USER=$USER
+export ACCOUNT=VERF-DEV
+export QUEUE=dev
+export QUEUESHARED=dev_shared
+export QUEUESERV=dev_transfer
+export PARTITION_BATCH=
+export nproc=128
+export USE_CFP=YES
+export WGRIB2=`which wgrib2`
+#export cyc=%CYC
+export cyc=00
+export NET=evs
+export evs_ver=${evs_ver}
+export STEP=prep
+export COMPONENT=subseasonal
+export RUN=atmos
+export MODELNAME="gefs cfs"
+export gefs_ver=${gefs_ver}
+export cfs_ver=${cfs_ver}
+export OBSNAME="gfs ecmwf osi ghrsst umd nam"
+export gfs_ver=${gfs_ver}
+export ccpa_ver=${ccpa_ver}
+export obsproc_ver=${obsproc_ver}
+export COMOUT=/lfs/h2/emc/vpppg/noscrub/$USER/EVS_Data/$NET/$evs_ver/$STEP/$COMPONENT/$RUN
+
+export config=$HOMEevs/parm/evs_config/subseasonal/config.evs.subseasonal.prep
+
```

Comment:
Add a line here to define the override setting for $FIXevs:
`export FIXevs=/lfs/h2/emc/vpppg/noscrub/emc.vpppg/verification/EVS_fix`